bigtop-packages/src/rpm/zookeeper/SPECS/zookeeper.spec

```diff
@@ -179,50 +180,58 @@ bash %{SOURCE2} \
           --system-lib-dir=%{lib_dir}
 
 %if  %{?suse_version:1}0
-orig_init_file=%{SOURCE4}
+     orig_init_file=%{SOURCE4}
+     %__install -d -m 0755 $RPM_BUILD_ROOT/%{initd_dir}/
+     init_file=$RPM_BUILD_ROOT/%{initd_dir}/%{svc_zookeeper}
+     %__cp $orig_init_file $init_file
+     chmod 755 $init_file
 %else
-orig_init_file=%{SOURCE3}
+#orig_init_file=%{SOURCE3}
 %endif
```

Comment:
I think we can remove the conditional since SUSE is not supported now.

Reply:
Thank you. I'll remove SUSE-releated code.